app/estimate_square_size.go

```diff
@@ -202,3 +202,8 @@ func overEstimateMalleatedTxSize(txLen, blobSize int) int {
 	// actually malleating the tx
 	return appconsts.MalleatedTxBytes + appconsts.MalleatedTxEstimateBuffer + malleatedTxLen
 }
+
+func isValidSquareSize(squareSize, minSquareSize, maxSquareSize int) bool {
+	// Check that square size is with range
+	return squareSize < minSquareSize || squareSize > maxSquareSize
+}
```

Comment:
[blocking] this function returns `true` if the squareSize is invalid so the function name is inaccurate. Either the function name or the return statement should be updated.

Proposal:

```suggestion
func isValidSquareSize(squareSize, minSquareSize, maxSquareSize int) bool {
	return squareSize >= minSquareSize && squareSize <= maxSquareSize
}
```